lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -597,8 +601,8 @@ int amf_app_handle_pdu_session_response(
                &smf_ctx->selected_ambr.ul_ambr_unit,
                &smf_ctx->selected_ambr.ul_session_ambr);
 
-  memcpy(&smf_ctx->subscribed_qos_profile, &(pdu_session_resp->qos_list),
-         sizeof(smf_ctx->subscribed_qos_profile));
+  smf_ctx->qos_flow_list = pdu_session_resp->qos_flow_list;
```

Comment:
Let us copy the structure qos_flow_list_t using memcpy

Reply:
> Let us copy the structure qos_flow_list_t using memcpy

Done